jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java

```diff
@@ -14,6 +14,7 @@
 
 import static com.epam.jdi.light.common.Exceptions.exception;
 import static com.epam.jdi.light.driver.get.DownloadDriverManager.*;
+import static com.epam.jdi.light.driver.get.DriverData.DEFAULT_DRIVER;
```

Comment:
Please revert all changes in this file.
None of them required for mobile

Reply:
All changes are required for mobile.

Reply:
nope )

Reply:
reverted